Makefile

```diff
@@ -74,11 +85,11 @@ coverage:
 lint: flake8 pylint
 
 flake8:
-	flake8 --ignore=$(FLAKE8_IGNORE) $(NAME)/ --max-line-length=80
+	flake8 --ignore=$(FLAKE8_IGNORE) --exclude slo_generator/exporters/gen/ $(NAME)/ --max-line-length=80
```

Comment:
can you please move the `slo_generator/exporters/gen` folder to `slo_generator/proto` so that we can split it from the rest of the exporters code ?